staging/publishing/rules.yaml

```diff
@@ -1922,9 +1922,18 @@ rules:
       branch: release-1.24
       dir: staging/src/k8s.io/pod-security-admission
   library: true
+- destination: noderesourcetopology-api
+  branches:
+  - name: master
+    dependencies:
+    - repository: code-generator
+      branch: master
+      dir: staging/src/k8s.io/noderesourcetopology-api
+    source:
+       branch: master
+       dir: staging/src/k8s.io/noderesourcetopology-api
+  library: true
 recursive-delete-patterns:
-- BUILD
-- '*/BUILD'
```

Comment:
is that chunk intentional?

Reply:
Ah, I think this was a mistake. Let me fix it. Thanks for that @kad.